pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -374,10 +453,14 @@ def build_model(
                 var=logistic_saturation(x=channel_adstock, lam=lam),
                 dims=("date", "channel"),
             )
+
+            channel_contributions_var = channel_adstock_saturated * beta_channel
+            if self.time_varying_media_effect:
+                channel_contributions_var *= tv_multiplier_media[:, None]
             channel_contributions = pm.Deterministic(
-                name="channel_contributions",
-                var=channel_adstock_saturated * beta_channel,
-                dims=("date", "channel"),
+                name="channel_contributions", 
+                var=channel_contributions_var, 
+                dims=("date", "channel"), 
             )
 
             mu_var = intercept + channel_contributions.sum(axis=-1)
```

Comment:
Since the HSGP is in dimension "date" from the beginning, I feel that we are first transforming said HSGP to a form that is compatible with `("date", "channel")` to return at the end to `"date"` only. Isn't it better to transform the channels to `"date"` and finally multiply by the HSGP without needing of `[:,None]`?

It's just an opinion, I think it would be more appropriate the other way and we can avoid a somewhat unnecessary transformation, what do you think?

Reply:
Don't fully understand. You can just commit the change if you believe it is best.

In general this time_varying_prior function supports two dimensions but I'm only using the 1D case here, maybe that's confusing.

I should actually add support for individually varying parameters in this PR too... not complicated

Reply:
> I should actually add support for individually varying parameters in this PR too... not complicated
I suggest we keep the scope of this PR small and then iterate